### PR TITLE
feat(nook1208): Jan 2nd plan completed

### PR DESCRIPTION
**Pull request by nook1208**
Signed-off-by: Sunwook Eom <sunwook5492@gmail.com>

## 완료된 목표 3가지를 적어주세요

1. [Rust-os: Chap6: bsp: device_driver: Add bcm module](https://github.com/nook1208/rust-rasp-os/commit/858c59baf4b8b5f2cf805641e91e36b6102f0380)
2. [English: Friends shadowing & Memorize sentences](https://github.com/nook1208/english/commit/7458eff723dc6ed653b794c95401abc91c9a97af)
3. [English: Write diaries](https://github.com/nook1208/english/commit/c5c273651b74dcce0a435ee4076d1751c097f960)

## 다음주 목표 3가지를 적어주세요

1. Rust-os: Write PL011UartInner struct and impl
2. English: Write diaries
3. Leet-code: Write a small program

### CODE REVIEW

**Comment by hayoung0Lee** [comment]
멋진 한주를 보내셨군요!!!!

**Review by hayoung0Lee — APPROVED** [review]
멋지십니다!!!!!

File: 2021/01/10/nook1208/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## 이번주 목표
+
+1. [Rust-os: Chap6: bsp: device_driver: Add bcm module](https://github.com/nook1208/rust-rasp-os/commit/858c59baf4b8b5f2cf805641e91e36b6102f0380)
+2. [English: Friends shadowing & Memorize sentences](https://github.com/nook1208/english/commit/7458eff723dc6ed653b794c95401abc91c9a97af)
+3. [English: Write diaries](https://github.com/nook1208/english/commit/c5c273651b74dcce0a435ee4076d1751c097f960)
```
Comment (juheekimm) [inline 2021/01/10/nook1208/README.md:5]: 꾸준함 👍 👍  멋져요
올 한해도 화이팅입니다! ㅎㅎ

**Review by juheekimm — APPROVED** [review]

**Review by 1ilsang — APPROVED** [review]
LGTM 고생하셨습니다.

**Pull request merged**